lib/ValidateDirectiveVisitor.ts

```diff
@@ -568,16 +588,21 @@ const wrapFieldResolverValidateArgument = <TContext>(
 
   const { resolve = defaultFieldResolver } = field;
   // eslint-disable-next-line no-param-reassign
-  field.resolve = function (...args): Promise<unknown> {
+  field.resolve = function (
+    source: unknown,
+    directiveArguments: AnyObject,
+    context: TContext,
+    info: GraphQLResolveInfo,
```

Comment:
do we need to change this?

Reply:
I had made this change to make it easier to remember the resolve function arguments and its order. But know we need it to create the `fieldResolveParameters`

Reply:
@luisfvieirasilva problem with this approach is that you break the code if the parameters change. It's not uncommon to get changes like an extra parameter... when you wrap stuff, you must do it in a safe way.

also, nobody has to remember this one, just need care and check the docs when doing it.

Reply:
Excellent point about the parameters changes, I'll fix it right away

Reply:
Done!